tests/sentry/backup/test_models.py

```diff
@@ -217,6 +228,74 @@ def test_organization(self):
         self.create_organization(owner=user)
         return self.import_export_then_validate()
 
+    @targets_models(Actor)
+    def test_actor(self):
+        self.create_user(email="test@example.com")
+        self.create_team(name="pre save team", organization=self.organization)
+        return self.import_export_then_validate()
+
+    @targets_models(ApiAuthorization)
+    def test_apiAuthorization(self):
+        user = self.create_user()
+        app = ApiApplication.objects.create(name="test", owner=user)
+        ApiAuthorization.objects.create(
+            application=app, user=self.create_user("example@example.com")
+        )
+        return self.import_export_then_validate()
+
+    @targets_models(ApiApplication)
+    def test_apiApplication(self):
+        user = self.create_user()
+        ApiApplication.objects.create(
+            owner=user, redirect_uris="http://example.com\nhttp://sub.example.com/path"
+        )
+        return self.import_export_then_validate()
+
+    @targets_models(ApiToken, ApiApplication)
```

Comment:
nit: I think we should avoid having multiple `@targets_models` entries for a single model if possible (`ApiApplication` seems to appear twice). That way we have a single "source of truth" for each model. This may not always be possible if we add more tests for different variations on models in the future, but at least for a first pass I think its a decent goal.